rethinking-css.md

```diff
@@ -30,104 +30,104 @@ Part of the problem is that CSS is so easy, that everyone on the team, regardles
+Teams without dedicated UI Engineers also often reach for some kind pre-fabricated CSS UI framework, such as Bootstrap or Semantic UI, to get them building and realising UI quickly. While the initial development period is promising (foregoing the fact you’re now serving to users 200 colours and 20 components you’ll never use), bloat continues to worsen as you encounter the next problem…
 
 ## Specificity wars
 
-Specialist UI Engineers have no issue understanding how CSS inheritance works, and also the 3 pillars of the "Cascading" in CSS: importance, specificity and source order. However, any or all of these can and often trip up less familiar developers.
+Specialist UI Engineers have no issue understanding how CSS inheritance works, nor the three pillars of the CSS “Cascade”: importance, specificity and source order. However, any or all of these can and often do trip up less familiar developers.
 
-As you add new features and styles, do they sometimes conflict with existing ones in surprising ways? Do you hack its specificity to get it to work by adding `!important`, an inline `style` or a more "qualified", nested selector? How does this affect the feature you build in 3 months' time? Do you then have to do the same to combat the last hack? It's a never-ending, losing battle.
+As you add new features and styles do they sometimes conflict with existing ones in surprising ways? Do you hack its specificity to get it to work by adding `!important`, an inline `style` or a more “qualified”, nested selector? How does this affect the feature you build in three months’ time? Do you then have to do the same to combat the last hack? It’s a never-ending, losing battle.
 
-Do you know how to calculate specificity? I've been a professional front-end developer since 2002 and I would not be able to reliably calculate it.
+Do you know how to calculate specificity? I’ve been a professional front-end developer since 2002 and I would not be able to reliably calculate it.
 
-But what if we never need to worry about it? What if we could essentially reduce CSS to a meaningful source order (that you're rarely aware of) and UI state helpers that are guaranteed to trump existing styles?
+But what if we never need to worry about it? What if we could essentially reduce CSS to a meaningful source order (that you’re rarely aware of) and UI state helpers that are guaranteed to trump existing styles?
 
 ## Naming things is hard
 
-When BEM was introduced to the front-end community, it was a revelation. Ugly at first, but a revelation nonetheless. Within CSS's global context, it allowed us to clearly express the hierarchy of components: its variants and their children.
+When BEM was introduced to the front-end community it was a revelation. Ugly at first, but a revelation nonetheless. Within CSS’s global context, it allowed us to clearly express the hierarchy of components: its variants and their children.
```

Comment:
Revert comma removal after "community". There's a natural pause that should be communicated

Reply:
Reverted. I could well be wrong, but I think some spoken pauses don't necessarily translate to a written comma.